velox/expression/fuzzer/tests/ArgumentTypeFuzzerTest.cpp

```diff
@@ -239,7 +239,7 @@ TEST_F(ArgumentTypeFuzzerTest, unsupported) {
           .argumentType("decimal(b_precision, b_scale)")
           .build();
 
-  testFuzzingFailure(signature, DECIMAL(13, 6));
+  testFuzzingFailure(signature, ROW({ARRAY(DECIMAL(13, 6))}));
```

Comment:
The return type in the signature is defined as `row(array(decimal(r_precision, r_scale)))`, so we need to make them aligned to test the fuzzing failure on decimal type. And without below fix, the fuzzing succeeds.
https://github.com/facebookincubator/velox/pull/9149/files#diff-80a484344bb5ae6cc0e32f0c0055707944cbe8a3a76b75feb47c825ae269bbbdR165-R171